developer/src/kmc-ldml/src/compiler/keymanweb-compiler.ts

```diff
@@ -22,7 +22,7 @@ export class LdmlKeyboardKeymanWebCompiler {
     const vk: VisualKeyboard.VisualKeyboard = vkc.compile(source);
 
     let result =
-      `{F: '${vk.header.unicodeFont.size}pt ${JSON.stringify(vk.header.unicodeFont.name)}', `+
+      `{F: ' 1em ${JSON.stringify(vk.header.unicodeFont.name)}', `+
```

Comment:
We always ignore font size in the vk export to kmw... so `1em` it is. (Fortunately, the whole font string is normally ignored and will be removed in a future version)